integrations/opsgenie/integration.py

```diff
@@ -110,11 +110,16 @@ class TeamResourceConfig(ResourceConfig):
     selector: TeamSelector
 
 
+class CommentResourceConfig(ResourceConfig):
+    kind: Literal["comment"]
+
```

Comment:
I think we need to support similar filters on alerts like we are doing in alert kind